src/tests/test_login.py

```diff
@@ -1,42 +1,35 @@
 import pytest
 from copy import deepcopy
 
+from app import db
 from app.api import crud, security
 from app.api.routes import login
-
+from tests.conf_test_db import get_entry_in_db, populate_db
 
 ACCESS_TABLE = [
     {"id": 1, "login": "first_login", "hashed_password": "first_pwd_hashed", "scopes": "me"},
     {"id": 2, "login": "second_login", "hashed_password": "second_pwd_hashed", "scopes": "me"},
 ]
 
-
-def _patch_session(monkeypatch, mock_table):
-    # DB patching
-    monkeypatch.setattr(login, "accesses", mock_table)
-    # Sterilize all DB interactions through CRUD override
-    monkeypatch.setattr(crud, "fetch_one", pytest.mock_fetch_one)
-    # Password
-    monkeypatch.setattr(security, "verify_password", pytest.mock_verify_password)
-
-
 @pytest.mark.parametrize(
     "payload, status_code, status_detail",
     [
         [{"username": "foo"}, 422, None],
         [{"password": "foo"}, 422, None],
         [{"username": "unknown", "password": "foo"}, 400, "Invalid credentials"],  # unknown username
         [{"username": "first", "password": "second"}, 400, "Invalid credentials"],  # wrong pwd
-        [{"username": "first_login", "password": "first_pwd"}, 200, None],  # valid
+        [{"username": "first_login", "password": "first_pwd"}, 200, None],  # should throw an error !!valid
```

Comment:
Why should it throw an error?

Reply:
Comment I forgot to delete. At first I misunderstood what this function was doing.